assets/lang/sklmessages_vi_VN.properties

```diff
@@ -0,0 +1,103 @@
+lang.name=Tiếng Việt
+lang.code=vi_VN
+
+tab.news=Thông tin
+tab.log=Log trình chạy
+tab.output=Game Output (%s)
+tab.crash=Báo cáo lỗi (%s)
+tab.loading=Đang tải...
+tab.loading.long=Trang tải quá lâu?
+tab.loading.refresh=Tải lại
+
+sidebar.switch=Đổi tài khoản
+sidebar.new=Thêm hồ sơ
+sidebar.edit=Chỉnh sửa hồ sơ
+sidebar.browse=Mở thư mục chứa hồ sơ
+sidebar.delete=Xóa hồ sơ
+sidebar.profiles=Hồ sơ đã có
+sidebar.loading=Đang tải hồ sơ...
+sidebar.settings=Cài đặt trình chạy
+sidebar.player=Người chơi
+sidebar.guest=Chào mừng, khách! Hãy đăng nhập.
+sidebar.versions=Đang tạo tài khoản...
+sidebar.ready.play=Sẵn sàng chơi
+sidebar.ready.update=Sẵn sàng để cập nhật và chơi
+sidebar.ready.download=Sẵn sàng để tải và chơi
+
+game.status.play=Chơi
+game.status.play.demo=Chơi thử
+game.status.play.offline=Chơi ngoại tuyến
+game.status.playing=Đang chơi...
+game.status.already=Đã đang chơi...
+game.status.loading=Đang tải...
+game.status.preparing=Đang chuẩn bị...
+game.status.downloading=Đang tải...
+game.status.installing=Đang cài đặt...
+game.status.launching=Đang mở game...
+game.status.idle=Không hoạt động
+
+game.output.open=Mở bảng điều khiển nổi
+game.output.lines=dòng được hiện
+
+game.conflict.title=Cảnh cáo mở lặp trình
+game.conflict.header=Bạn đã mở một trình Minecraft rồi.
+game.conflict.text=Nếu bạn mở thêm một trình nữa trong cùng thư mục, nó có thể gây lỗi và làm hỏng save của bạn. \nNó có thể gây ra nhiều lỗi trong chơi đơn và chơi mạng và khác. Chúng tôi sẽ không chịu trách nhiệm cho gì xảy ra. \nBạn vẫn muốn chạy thêm một trình Minecraft nữa không dù điều này có thể xảy ra? \nBạn có thể tránh lỗi này bằng cách mở Minecraft ở thư mục khác (xem phần "Chỉnh sửa hồ sơ").
+
+login.username=Email của Mojang
+login.username.offline=Tài khoản
+login.password=Mật khẩu
+login.offline=Ngoại tuyến
+login.login=Đăng nhập
+login.existing.name=Tải khoản đã đăng nhập
+login.existing.play=Chơi
+login.existing.logout=Đăng xuất
+
+editor.profilename=Tên tài khoản
+editor.copy=Bản sao lưu của %s
+editor.gamedir=Thư mục game
+editor.resolution=Độ phân giải
+editor.visibility=Cách trình chạy hiển thị
+editor.hide=Ẩn trình chạy và mở lại khi game kết thúc
+editor.close=Đóng trình chạy khi game mở
+editor.keep=Giữ trình chạy mở
+editor.snap=Snapshots
+editor.snap.tooltip=Cho phéo sử dụng bản thử nghiệm ("snapshots")
+editor.beta=Beta
+editor.beta.tooltip=Cho phép sử dụng Minecraft "Beta" (Từ 2010–2011)
+editor.alpha=Alpha
+editor.alpha.tooltip=Cho phép sử dụng Minecraft "Alpha"(Từ 2010)
```

Comment:
You have probably forgotten to insert a whitespace after the word "Alpha" :)

Reply:
Yea I guess I didn't check it as carefully as I think. Can you edit it for me?

Reply:
I don't have the ability to modify anything in your fork, because you haven't given me permissions for doing this.